lib/Fmt_odoc.ml

```diff
@@ -203,7 +203,9 @@ and fmt_list_light c kind items =
 
 and fmt_nestable_block_elements c ?(prefix = noop) = function
   | [] -> noop
-  | elems -> prefix $ list_block_elem elems (fmt_nestable_block_element c)
+  | elems ->
+      prefix
+      $ hovbox 0 (list_block_elem elems (fmt_nestable_block_element c))
```

Comment:
That will double-box list elements. Can this be an issue ?
Otherwise, we can move the box and the prefix logic into `fmt_tag_args`.

Reply:
Good catch, should be fixed now!